kits/youtube-creator-copilot/apps/app/globals.css

```
@@ -0,0 +1,560 @@
+@tailwind base;
```

Stylelint (17.14.0)

[error] Unknown rule scss/at-rule-no-unknown. Did you mean at-rule-no-unknown?

(scss/at-rule-no-unknown)